src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -174,6 +162,43 @@ export const mapColonyEventToExpectedFormat = (
   };
 };
 
+export const mapMotionEventToExpectedFormat = (
+  actionData: ColonyAction,
+  initiatorUser?: User | null,
+  motionMessageData?: MotionMessage,
+) => {
+  return {
+    eventNameDecorated: <b>{motionMessageData?.name}</b>,
+    amountTag: (
+      <AmountTag>
+        <Numeral
+          value={motionMessageData?.amount ?? 0}
+          decimals={actionData.token?.decimals ?? undefined}
```

Comment:
The token might not be present on the actionData, as it's an optional field in the schema and will only get included if it's a token-related motion. 

However, it would be the same value as the token you can get from colony context, because we just take it from the colony client in the block ingestor. So here, to ensure you also have the decimals/symbol, you can just take it off the colony object, and you don't need to pass down `actionData`.

Reply:
Got it, Im assuming you are referring to the nativeToken